sljit_src/sljitProtExecAllocator.c

```diff
@@ -145,7 +146,7 @@ static SLJIT_INLINE int create_tempfile(void)
 	}
 
 #ifdef O_TMPFILE
```

Comment:
I have been thinking whether we should make this flag mandatory? Or drop this part?

Reply:
O_TMPFILE is Linux specific, so if mandatory will break NetBSD
as far as memfd_create is not mandatory then using this is a better option than going all the way to mkostemp so discarding it wouldn't be recommended either IMHO